app/helpers/dfe_signin_helper.rb

```diff
@@ -0,0 +1,9 @@
+module DfeSigninHelper
+  def dfe_signin_user_audit_link(user)
+    "https://support.signin.education.gov.uk/users/#{user.sign_in_user_id}/audit"
```

Comment:
Any reason not to put the hostname into `Settings`?

Reply:
Removed